main/main.cpp

```diff
@@ -4051,7 +4056,7 @@ int Main::start() {
 				sml->get_root()->set_embedding_subwindows(true);
 			}
 			restore_editor_window_layout = EditorSettings::get_singleton()->get_setting(
-					"interface/editor/remember_window_size_and_position");
+												   "interface/editor/editor_screen") == -5;
```

Comment:
How do I compare this variant to this signed int?

Reply:
You cast it to an int, like `int(EditorSettings::get_singleton()->get_setting("interface/editor/editor_screen"))` or `EditorSettings::get_singleton()->get_setting("interface/editor/editor_screen").operator int()`

Reply:
I'm a total beginner with c++, is there a convention of using the raw values over something like an enum here?

Reply:
You should use the enum to make it safer

Reply:
Okay, I found the enums! lmk if i'm doing this right